src/NewTools-SettingsBrowser/StSettingPresenterItem.class.st

```diff
@@ -26,6 +34,12 @@ StSettingPresenterItem >> defaultLayout [
 		yourself
 ]
 
+{ #category : 'initialization' }
+StSettingPresenterItem >> environment [
+
+	^ SystemEnvironment new
```

Comment:
If you create a new environment, when we remove the singleton of the SystemAnnouncer, nobody will get those announcements